tools/bin/hawqregister

```diff
@@ -40,119 +38,126 @@ EXECNAME = os.path.split(__file__)[-1]
 setup_tool_logging(EXECNAME,getLocalHostname(),getUserName())
 
 
-def create_opt_parser(version):
+def option_parser():
     parser = OptParser(option_class=OptChecker,
                        usage='usage: %prog [options] database_name table_name file_or_dir_path_in_hdfs',
-                       version=version)
+                       version='%prog version $Revision: #1 $')
     parser.remove_option('-h')
     parser.add_option('-?', '--help', action='help')
-    parser.add_option('-h', '--host', help="host of the target DB")
-    parser.add_option('-p', '--port', help="port of the target DB", type='int', default=0)
-    parser.add_option('-U', '--user', help="username of the target DB")
-    return parser
+    parser.add_option('-h', '--host', help='host of the target DB')
+    parser.add_option('-p', '--port', help='port of the target DB', type='int', default=0)
+    parser.add_option('-U', '--user', help='username of the target DB')
+    parser.add_option('-d', '--database', default = 'postgres', dest = 'database', help='database name')
```

Comment:
@xunzhang , I think you can put the interface onto JIRA description, e.g. the parameters description. Thanks